packages/contracts-bedrock/scripts/getting-started/config.sh

```diff
@@ -65,7 +65,7 @@ config=$(cat << EOL
   "l1FeeVaultWithdrawalNetwork": 0,
   "sequencerFeeVaultWithdrawalNetwork": 0,
 
-  "gasPriceOracleOverhead": 2100,
+  "gasPriceOracleOverhead": 0,
```

Comment:
wouldn't this break? 
https://github.com/ethereum-optimism/optimism/blob/93891de2176f6f997da80a4de8de84466164bb25/op-node/rollup/types.go#L281-L286

Reply:
https://github.com/ethereum-optimism/optimism/blob/93891de2176f6f997da80a4de8de84466164bb25/op-chain-ops/genesis/config.go#L639

a 0 Overhead will turn into:
```
SystemConfig: eth.SystemConfig{
				BatcherAddr: d.BatchSenderAddress,
				Overhead:    eth.Bytes32(common.BigToHash(new(big.Int).SetUint64(d.GasPriceOracleOverhead))),
				Scalar:      eth.Bytes32(d.FeeScalar()),
				GasLimit:    uint64(d.L2GenesisBlockGasLimit),
			},
```
empty 32 bytes.

After which op-node validation fails:
https://github.com/ethereum-optimism/optimism/blob/58a3fde0a00f00ba2a10ac84e3bfe960eb585c55/op-node/rollup/types.go#L262-L264

Reply:
Thanks for highlighting. This will be addressed here https://github.com/ethereum-optimism/optimism/pull/10226